hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1172,6 +1172,22 @@ message KeyInfo {
     optional uint64 expectedDataGeneration = 22;
 }
 
+message KeyInfoProtoLight {
```

Comment:
add comment for this type specifying its sub-part of keyInfo with selected fields only and keeping index same.

Reply:
done.